jest.setup.js

```diff
@@ -16,7 +16,7 @@ beforeAll(async () => {
   };
 
   await mongoose.connect(uri, mongoOptions);
-});
+}, 4000);
```

Comment:
the default timeout is 5000 so i think you should remove the `, 4000` and just keep the `});` on line 19